<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>window_main</class>
 <widget class="QMainWindow" name="window_main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>994</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Super Mario World Editor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::South</enum>
      </property>
      <property name="currentIndex">
       <number>3</number>
      </property>
      <property name="documentMode">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="tab_level_editor">
       <attribute name="title">
        <string>Level Editor</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_gfx_editor">
       <attribute name="title">
        <string>GFX Editor</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_rats_viewer">
       <attribute name="title">
        <string>RATS Tags Viewer</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2"/>
      </widget>
      <widget class="QWidget" name="tab_level_list_viewer">
       <attribute name="title">
        <string>All Levels</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>994</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuExpand_ROM">
     <property name="title">
      <string>Expand ROM</string>
     </property>
     <addaction name="actionExpand_to_1MB"/>
     <addaction name="actionExpand_to_2MB"/>
     <addaction name="actionExpand_to_4MB"/>
    </widget>
    <addaction name="action_menu_open_rom"/>
    <addaction name="action_menu_save_rom"/>
    <addaction name="separator"/>
    <addaction name="menuExpand_ROM"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionGFX_editor"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionContents"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuPlugins">
    <property name="title">
     <string>Plugins</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuPlugins"/>
   <addaction name="menuWindow"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>Toolbar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="tool_select_level"/>
   <addaction name="separator"/>
   <addaction name="tool_select_layer_1"/>
   <addaction name="tool_select_layer_2"/>
   <addaction name="tool_edit_sprites"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_menu_open_rom">
   <property name="text">
    <string>Open ROM</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="tool_select_level">
   <property name="text">
    <string>level</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Select level number in ROM. Range 000-1FF.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="tool_select_layer_1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>layer 1</string>
   </property>
   <property name="toolTip">
    <string>Edit layer 1 level data.</string>
   </property>
  </action>
  <action name="tool_select_layer_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>layer 2</string>
   </property>
   <property name="toolTip">
    <string>Edit layer 2 level data.</string>
   </property>
  </action>
  <action name="tool_edit_sprites">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>sprites</string>
   </property>
   <property name="toolTip">
    <string>Edit sprite level data.</string>
   </property>
  </action>
  <action name="actionGFX_editor">
   <property name="text">
    <string>GFX editor</string>
   </property>
  </action>
  <action name="actionContents">
   <property name="text">
    <string>Contents</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionEditors">
   <property name="text">
    <string>Editors</string>
   </property>
  </action>
  <action name="action_menu_save_rom">
   <property name="text">
    <string>Save ROM</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExpand_to_1MB">
   <property name="text">
    <string>Expand to 1MB</string>
   </property>
  </action>
  <action name="actionExpand_to_2MB">
   <property name="text">
    <string>Expand to 2MB</string>
   </property>
  </action>
  <action name="actionExpand_to_4MB">
   <property name="text">
    <string>Expand to 4MB</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
